public/admin/index.html

```
@@ -1116,6 +1128,68 @@ <h4>Recent deliveries</h4>
+                <label class="form-inline">
+                  <span>Title</span>
+                  <input type="text" id="notifBroadcastTitle" maxlength="80" required autocomplete="off" />
+                </label>
+                <label class="form-inline">
+                  <span>Body</span>
+                  <input type="text" id="notifBroadcastBody" maxlength="200" required autocomplete="off" />
+                </label>
+                <label class="form-inline">
+                  <span>URL (site-relative)</span>
+                  <input
+                    type="text"
```

HTMLHint (1.9.2)

[warning] No matching [ label ] tag found.

(input-requires-label)